recipes/resiprocate/cmake/conanfile.py

```diff
@@ -0,0 +1,123 @@
+            self.requires("openssl/[>=1.1 <4]")
+
+    def validate(self):
+        check_min_cppstd(self, 11)
+        if is_msvc(self) and self.options.shared:
+            raise ConanInvalidConfiguration(f"{self.ref} can not be built as shared on Visual Studio and msvc.")
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.21 <4]")
+        if not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
+            self.tool_requires("pkgconf/[>=2.2 <3]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_DSO_PLUGINS"] = False
+        tc.variables["BUILD_QPID_PROTON"] = False
+        tc.variables["BUILD_RECON"] = False
+        tc.variables["BUILD_REFLOW"] = False
+        tc.variables["BUILD_REND"] = False
+        tc.variables["BUILD_REPRO"] = False
+        tc.variables["BUILD_RETURN"] = False
+        tc.variables["BUILD_TESTING"] = False
+        tc.variables["BUILD_TFM"] = False
+        tc.variables["ENABLE_LOG_REPOSITORY_DETAILS"] = False
+        tc.variables["REGENERATE_MEDIA_SAMPLES"] = False
+        tc.variables["RESIP_ASSERT_SYSLOG"] = False
+        tc.variables["USE_CONTRIB"] = False
+        tc.variables["USE_DTLS"] = self.options.with_ssl
+        tc.variables["USE_KURENTO"] = False
+        tc.variables["USE_MAXMIND_GEOIP"] = False
+        tc.variables["USE_NUGET"] = False
+        tc.variables["USE_POPT"] = False
+        tc.variables["VERSIONED_SONAME"] = False
+        tc.variables["WITH_C_ARES"] = self.options.with_cares
+        tc.variables["WITH_SSL"] = self.options.with_ssl
+        if self.settings.os in ["Linux"]:
+            tc.preprocessor_definitions["RESIP_RANDOM_THREAD_LOCAL"] = True
+        if cross_building(self):
+            tc.cache_variables["HAVE_CLOCK_GETTIME_MONOTONIC"] = not self.settings.os in ["Windows"]
+        tc.generate()
+        tc = PkgConfigDeps(self)
+        tc.generate()
+        tc = CMakeDeps(self)
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "COPYING", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.la", os.path.join(self.package_folder, "lib"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "lib"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+
+    def package_info(self):
+        resiprocate_lib = "resiprocate" if self.settings.os == "Windows" else "resip"
+        self.cpp_info.libs = [resiprocate_lib, "rutil", "dum"]
```

Comment:
@AbrilRBS is working on this to split these libraries into several components. She'll update the PR soon